_studio/mfx_lib/encode_hw/h264/src/mfx_h264_encode_hw_utils.cpp

```diff
@@ -3569,7 +3569,7 @@ void MfxHwH264Encode::PrepareSeiMessageBuffer(
         needBufferingPeriod = needPicTimingSei = 0; // in CQP HRD mode application inserts BP and PT SEI itself
 
     mfxU32 needAtLeastOneSei =
-        (task.m_ctrl.NumPayload > secondFieldPicFlag && task.m_ctrl.Payload[secondFieldPicFlag] != 0) ||
+        (task.m_ctrl.NumPayload > secondFieldPicFlag && task.m_ctrl.Payload && task.m_ctrl.Payload[secondFieldPicFlag] != 0) ||
```

Comment:
from line 3627 I see that task.m_ctrl.Payload[secondFieldPicFlag] is a pointer, so use nullptr for comparison